tools/cdk-build-tools/config/nycrc

```diff
@@ -6,6 +6,7 @@
     "branches": 50,
     "reporter": [
         "html",
+        "lcov",
```

Comment:
?

Reply:
This enables use of the [Coverage Gutters](https://marketplace.visualstudio.com/items?itemName=ryanluker.vscode-coverage-gutters) plugin in Visual Studio code to determine what is (or is not) covered, which editor highlights updating "real-time" as you re-run the tests.

Reply:
I've used that because my statement coverage dropped below 80% as a consequence of re-factoring the `export` implementation out of the `Base` class.

Reply:
Oooo shiny! installed